berkdb/os/os_rw.c

```diff
@@ -725,8 +725,8 @@ __os_physwrite(dbenv, fhp, addr, len, nwp)
 				else {
 					/* We got positioned into the limbo somewhere (how?)
 					 * so it's not safe to proceed. */
-					__db_err(dbenv, "lseek: fd %d off %lu, wanted %lu\n", fhp->fd,
-							rc, off);
+					__db_err(dbenv, "lseek: fd %d off %u, wanted %u\n", fhp->fd,
+							(unsigned int)rc, (unsigned int)off);
```

Comment:
`off_t` is signed value. cast to int64_t and print using pri64d?

Reply:
yep, thats good suggestion. thanks.